WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -184,6 +190,15 @@ import Foundation
             return "reader_toggle_follow_conversation"
         case .readerPostReported:
             return "reader_post_reported"
+        // What's New
+        case .whatIsNewShownOnAppUpdate:
+            return "whatsnew_shown_on_app_update"
```

Comment:
Hi @frosty ! Thanks for reviewing!!

> Can we match the events used on Android? 

we sure can!

> There we have these events:
> 
> FEATURE_ANNOUNCEMENT_SHOWN_ON_APP_UPGRADE
> FEATURE_ANNOUNCEMENT_SHOWN_FROM_APP_SETTINGS
> both send `feature_announcement_shown`
> FEATURE_ANNOUNCEMENT_FIND_OUT_MORE_TAPPED
> FEATURE_ANNOUNCEMENT_CLOSE_DIALOG_BUTTON_TAPPED
> both send `feature_announcement_button_tapped`.
> 
> The first two events have a property: `source` -> `app_upgrade` or `app_settings`.
> The second two events have a property: `button` -> `close_dialog` or `find_out_more`.

So, if I interpret this correctly, it would translate to iOS like so:

we would have two new `WPAnalyticsEvent` cases:

```
case featureAnnouncementShown
case featureAnnouncementButtonTapped
```

with the following values:

```
case .featureAnnouncementShown
    return "feature_announcement_shown"
case .featureAnnouncementButtonTapped
    return "feature_announcement_button_tapped"
```

then the properties would be set accordingly, depending if it's an app update or from App settings (for the modal presentation) or if it's the continue button to the find out more button. 

Is this correct?

Reply:
Yes, this sounds right to me!

Reply:
Done!!